lib/components/Snackbar/snackbar.scss

```diff
@@ -0,0 +1,97 @@
+@import "@quill/spacing.scss";
+@import "@quill/border.scss";
+@import "@quill/breakpoints.scss";
+@import "@quill/motion.scss";
+@import "@quill/elevation.scss";
+
+.snackbar {
+    display: flex;
+    width: 340px;
+    max-width: 560px;
+    min-height: 48px;
+    padding: var(--semantic-spacing-general-sm)
+        var(--semantic-spacing-general-md) var(--semantic-spacing-general-sm);
+    align-items: center;
+    gap: var(--semantic-spacing-gap-md);
+    border-radius: var(--semantic-borderRadius-md);
+    border: var(--borderWidth-xs) solid
+        var(--core-color-opacity-white-100, rgba(255, 255, 255, 0.08));
+    background: var(--core-color-solid-slate-1400, #000);
+    box-shadow: var(--core-elevation-shadow-330);
+    position: absolute;
+    z-index: 1;
+    bottom: -100%;
+    transition: bottom var(--motion-duration-snappy)
+        var(--motion-easing-inandout);
+
+    &.slow-animation {
+        animation:
+            slide-up var(--motion-duration-snappy) var(--motion-easing-inandout)
+                forwards,
+            slide-down var(--motion-duration-snappy) 3.5s
+                var(--motion-easing-inandout) forwards;
+    }
+    &.fast-animation {
+        animation:
+            slide-up var(--motion-duration-snappy) var(--motion-easing-inandout)
+                forwards,
+            slide-down var(--motion-duration-snappy) 1.5s
+                var(--motion-easing-inandout) forwards;
+    }
+
+    @include breakpoint("sm") {
```

Comment:
When you use this mixin do you receive this warning?
`Warning: Unknown breakpoint: sm`

Reply:
@jia-deriv where should i be seeing? localhost or storybook? Not seeing anything

Reply:
Something like this when I use it `@include breakpoint("lg") {
        display: none;
    }`
![image](https://github.com/deriv-com/quill-ui/assets/142988136/d1cc411c-a80c-4e33-92b7-d8ed22ff56f8)

Reply:
did you import the breakpoint file into your scss?

@jia-deriv

Reply:
Some condition issue in the breakpoint mixin should be fix later.